### DISCUSSION TITLE
Dev question

### DISCUSSION THREAD

**Discussion by 3djc** [creepymonster/GlucoseDirect#583] [Q&A]
I have never dev on Apple platform, but I'm thinking about it for GD.

If i buy a Mac and I try some changes and compile with Xcode, can I upload that to my phone/watch without have a Apple developer account ? I have no interest in ever putting something on the store, but I love love to try helping with GD

**Comment by henkjanh** [comment]
I had the same question and my experience is that you will need a paid Apple Developer account to be able to compile the "Near Field Communication Tag Reading" and "Time Sensitive Notifications" capabilities. Please see my post about it: https://github.com/creepymonster/GlucoseDirect/discussions/543

Of course, if you're not planning to actually use the app on your phone/watch you could still work with it by disabling these capabilities and deploy to your phone/watch. Not sure if that would interfere with testing though (I'm not an app developer).

**Reply by 3djc** [reply]
Crap, need to buy a Mac, and then 99€ / year, starts to be expensive just to contribute to GD.

Thanks for your extensive answer tho :)
